co-authors-plus.php

```diff
@@ -1050,24 +1050,112 @@ function filter_wp_get_object_terms( $terms, $object_ids, $taxonomies, $args ) {
 	function filter_count_user_posts( $count, $user_id ) {
 		$user = get_userdata( $user_id );
 
-		if ( is_object( $user ) ) {
+		// Return combined post count, if account is linked.
+		if ( $this->has_linked_account( $user ) ) {
```

Comment:
Introducing a new helper function won't be necessary for this, as I'm seeing that all it does is return the boolean of `get_coauthor_by()` and on line 1060, `get_coauthor_by()` is also being called. For better readability, is there a better way we could just call the function once? See DRY principle: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
I see what you mean. Initially, I introduced `has_linked_account()` as it made that part easier to read for me. That said, I updated that part to avoid DRY.